sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/sample_convert_to_and_from_dict.py

```diff
@@ -0,0 +1,119 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_convert_to_and_from_dict.py
+
+DESCRIPTION:
+    This sample demonstrates how to convert models returned from an analyze operation
+    to and from a dictionary. The dictionary in this sample is then converted to a
+    JSON file, then the same dictionary is converted back to its original model.
+
+USAGE:
+    python sample_convert_to_and_from_dict.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer API key
+"""
+
+import os
+import json
+
+def convert_to_and_from_dict():
+    path_to_sample_documents = os.path.abspath(
+        os.path.join(
+            os.path.abspath(__file__),
+            "..",
+            "..",
+            "./sample_forms/forms/Form_1.jpg",
+        )
+    )
+
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.formrecognizer import DocumentAnalysisClient, AnalyzeResult
+
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+
+    document_analysis_client = DocumentAnalysisClient(
+        endpoint=endpoint, credential=AzureKeyCredential(key)
+    )
+    with open(path_to_sample_documents, "rb") as f:
+        poller = document_analysis_client.begin_analyze_document(
+            "prebuilt-document", document=f
+        )
+    result = poller.result()
+
+    # connvert the received model to a dictionary
+    d = result.to_dict()
+
+    # save the dictionary as a JSON content in a JSON file
+    with open('data.json', 'w') as f:
+        json.dump(d, f)
+
```

Comment:
This is a key part of this sample to show how one of our samples can be saved to a JSON file

Reply:
The problem is that it doesnt delete the file...I'm also good changing this to converting the dict to a JSON string so we dont create files.

Reply:
I'm personally fine with it creating the file so someone can inspect and delete as they wish.